front-end-poc/src/Audit/types.ts

```diff
@@ -19,3 +19,18 @@ export interface IExecutionRouteParams {
     executionId: string,
     executionType: ExecutionType
 }
+
+export interface IFullModelResponse {
+    executionId: number,
```

Comment:
Do we include the executionId in the response? In addition to that, shouldn't be a string?

Reply:
Removed in new commit (I also made some properties optional as they can be `null` for now).